draft-ietf-quic-qlog-h3-events.md

```diff
@@ -508,6 +516,19 @@ H3UnknownFrame = {
 ~~~
 {: #h3unknownframe-def title="UnknownFrame definition"}
 
+## H3DatagramFrame Definition
+
+The HTTP/3 DATAGRAM frame is defined in {{Section 2 of !RFC9297}}.
+
+~~~ cddl
+H3DatagramFrame = {
+    frame_type: "datagram"
+    quarter_stream_id: uint64
```

Comment:
should this have its own type with max 2^60-1 ?
I guess we don't do that for other fields e.g. the ones encoded with a QUIC Integer

Reply:
actually thinking more about this, I think it's better to keep the full range. 
This way it would be easier to spot if an implementation is not using the id correctly